k8s/application/server/deployment.yaml

```diff
@@ -16,6 +16,15 @@ spec:
         app: server
     spec:
       containers:
+      - name: istio-proxy
+        image: auto
+        # istio-proxies are injected by anthos, see `istio.io/rev: asm-managed` in ../namespace.yaml
+        # these are overrides for the default configuration of those injected proxy sidecars
+        resources:
```

Comment:
If this doesn't work, another way to set this is using annotations: https://istio.io/latest/docs/reference/config/annotations/